blog/views.py

```diff
@@ -20,7 +20,6 @@ def post_new(request):
         if form.is_valid():
             post = form.save(commit=False)
             post.author = request.user
-            post.published_date = timezone.now()
```

Comment:
@webknjaz to save the post as a draft. New posts will be saved as drafts rather than being instantly published. When the draft post has been created, there is no need to know the exact time until this post will be publish.